docs/t-sql/functions/serverproperty-transact-sql.md

```diff
@@ -58,32 +58,36 @@ Is an expression that contains the property information to be returned for the s
 |IsIntegratedSecurityOnly|Server is in integrated security mode.<br /><br /> 1 = Integrated security (Windows Authentication)<br /><br /> 0 = Not integrated security. (Both Windows Authentication and [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] Authentication.)<br /><br /> NULL = Input is not valid, an error, or not applicable.<br /><br /> Base data type: **int**|  
-|IsLocalDB|**Applies to**: [!INCLUDE[ssSQL11](../../includes/sssql11-md.md)] and later.<br /><br /> Server is an instance of [!INCLUDE[ssExpress](../../includes/ssexpress-md.md)] LocalDB.<br /><br /> NULL = Input is not valid, an error, or not applicable.|  
+|IsLocalDB|**Applies to**: [!INCLUDE[ssSQL11](../../includes/sssql11-md.md)] and later.<br /><br /> Server is an instance of [!INCLUDE[ssExpress](../../includes/ssexpress-md.md)] LocalDB.<br /><br /> NULL = Input is not valid, an error, or not applicable.<br /><br /> Base data type: **int**|  
 |IsPolyBaseInstalled|**Applies to**: [!INCLUDE[ssCurrent](../../includes/sscurrent-md.md)].<br /><br /> Returns whether the server instance has the PolyBase feature installed.<br /><br /> 0 = PolyBase is not installed.<br /><br /> 1 = PolyBase is installed.<br /><br /> Base data type: **int**|  
 |IsSingleUser|Server is in single-user mode.<br /><br /> 1 = Single user.<br /><br /> 0 = Not single user<br /><br /> NULL = Input is not valid, an error, or not applicable.<br /><br /> Base data type: **int**|  
+|IsTempDbMetadataMemoryOptimized|**Applies to**: [!INCLUDE[ssSQL2019](../../includes/sssqlv15-md.md)] and later.<br /><br />Returns 1 if TempDB has been enabled to use memory-optimized tables for metadata; 0 if TempDB is using regular, disk-based tables for metadata. For more information, see [TempDB Database](../../relational-databases/databases/tempdb-database#memory-optimized-tempdb-metadata).<br /><br /> Base data type: **int**|  

```

Comment:
This looks fine, just request a change to the case of tempdb. We are standardizing on tempdb vs. TempDB, I've already made the changes to the tempdb Database article linked in your submission.

Reply:
Thanks Pam! I've updated this PR.